static/js/modules/tables.js

```diff
@@ -589,8 +589,9 @@ DataTable.prototype.fetch = function(data, callback) {
 
 DataTable.prototype.export = function() {
   var url = this.buildUrl(this.api.ajax.params(), false);
-  download.download(url);
+  var item = download.download(url);
   this.disableExport({message: DOWNLOAD_MESSAGES.pending});
+  item.$body.find('.button').focus();
```

Comment:
Well the problem is that we only want to move focus if the user has initiated a download by clicking export. If we included it in `DownloadItem.init` it would be set even when stored downloads are hydrated when loading a page, which would be disruptive. Or is there a better way to do this?